plugins/modules/aci_l3out_hsrp_secondary_vip.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_hsrp_secondary_vip
+short_description: Manage HSRP Secondary Virtual IP (hsrpSecVip) of a HSRP group (hsrpGroupP)
+description:
+- Manage HSRP Secondary Virtual IP of a HSRP group on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+    required: true
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+    required: true
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+    required: true
+  interface_profile:
+    description:
+    - Name of an existing interface profile.
+    type: str
+    aliases: [ name, interface_profile_name, logical_interface ]
+    required: true
+  hsrp_interface_group:
+    description:
+    - Name of an existing hsrp group.
+    type: str
+    aliases: [ name, hsrp_group ]
+    required: true
+  secondary_virtual_ip:
+    description:
+    - The version of the compatibility catalog.
+    type: str
+    aliases: [ vip, secondary_vip ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+notes:
+- The C(tenant), C(l3out), C(logical_node_profile), C(logical_interface_profile), C(hsrp_interface_profile) and C(hsrp_group) must exist before using \
+  this module in your playbook. The M(cisco.aci.aci_tenant), M(cisco.aci.aci_l3out), M(cisco.aci.aci_l3out_logical_node_profile), \
+  M(cisco.aci.aci_l3out_logical_interface_profile), M(cisco.aci.aci_l3out_hsrp_interface_profile) and M(cisco.aci.aci_l3out_hsrp_group) can be used for \
+  this.
+seealso:
+- module: aci_tenant
+- module: aci_l3out
+- module: aci_l3out_logical_node_profile
+- module: aci_l3out_logical_interface_profile
```

Comment:
hsrp_interface_profile not defined as input

Reply:
l3out_hsrp_interface_profile is present below. However, I'm assuming you meant hsrp_interface_policy in which case it's not required.

Reply:
perhaps I am not understanding properly:

C(tenant). -> tenant
C(l3out). -> l3out
C(logical_node_profile). -> node_profile
C(logical_interface_profile). -> interface_profile
C(hsrp_interface_profile). -> which input is this mapping to? Or it requires to be present but is no input?
C(hsrp_group). -> hsrp_group